src/ExtendedDateFilters.ts

```diff
@@ -77,20 +77,41 @@ export namespace ExtendedDateFilters {
         | IAbsoluteDateFilterForm
         | IAbsoluteDateFilterPreset;
 
-    export const isAbsoluteDateFilterOption = (option: DateFilterOption): option is AbsoluteDateFilterOption =>
+    export const isAllTimeDateFilter = (option: DateFilterOption): option is IAllTimeDateFilter =>
```

Comment:
Where is this used?

Reply:
In KD, for example [here](https://github.com/gooddata/gdc-dashboards/blob/develop/app/services/ExtendedDateFilterService.ts#L219)

Reply:
Makes sense